### PR TITLE
Like unlike

### PR DESCRIPTION
**Pull request by ngoctt-0375**
## Related Tickets
- [ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/67387)

## WHAT (optional)
- Like và unLike sách

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
[Screencast from 12-09-2023 16:50:08.webm](https://github.com/awesome-academy/ruby_naitei19_library_management/assets/142281496/97c82e43-15ef-4e80-a7ef-cfd205d43144)

### CODE REVIEW

**Comment by ngoctt-0375** [comment]
ready

File: app/views/like_books/_like_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div id="follow_form" class="d-flex flex-row bd-highlight mb-3">
+  <% if current_user.books.exists?(@book.id) %>
+    <%= render "like_books/unlike" %>
+    <div class="mt-4 px-4">
+      <%= t("like.you") + (@book.user_like_books.count - 1).to_s + t("like.people") if @book.user_like_books.count > 1%>
```
Comment (ngocvt-0484) [inline app/views/like_books/_like_form.html.erb:5]: viết helper nhé e. tránh viết logic ở view

Comment (ngocvt-0484) [inline app/views/like_books/_like_form.html.erb:5]: viết helper nhé e. tránh viết logic ở view

**Comment by ngoctt-0375** [comment]
ready

**Comment by manhtd-1083** [comment]
ready

**Pull request merged**